front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -518,11 +521,41 @@ export function MessageView({
           </div>
           <div
             className={classNames(
-              "break-word ml-2  flex flex-1 flex-col whitespace-pre-wrap pt-1",
+              "break-word relative  ml-2 flex flex-1 flex-col whitespace-pre-wrap pt-1",
               message.role === "user" ? "italic text-gray-500" : "text-gray-700"
             )}
           >
             {toMarkdown(message)}
+            {message.role === "assistant" ? (
+              <div
+                className="absolute -top-1.5 right-0 mt-2 hidden text-gray-400 hover:text-violet-800 group-hover:block"
```

Comment:
Why did you need to have the position absolute? (I'm not an expert 😬 )

Reply:
I need it outside the normal flow to position it at top right of the parent element

Reply:
not sure this explanation is clear :sweat_smile: we can discuss it more if you'd like when you want (belle île? :wink: